src/date/expand-pattern.js

```diff
@@ -24,8 +24,30 @@ define([
  * - { datetime: "full" } returns "EEEE, MMMM d, y 'at' h:mm:ss a zzzz";
  * - { pattern: "dd/mm" } returns "dd/mm";
  */
+
 return function( pattern, cldr ) {
-	var result;
+	var result, getDateTime, skeleton, dateSkeleton = "", timeSkeleton = "";
+
+	getDateTime = function( type, dateSkeleton, timeSkeleton, cldr ) {
+		var result;
+		result = cldr.main([
+			"dates/calendars/gregorian/dateTimeFormats",
+			type
+		]);
+		if ( result ) {
+			result = formatMessage( result, [
+				cldr.main([
+					"dates/calendars/gregorian/dateTimeFormats/availableFormats",
+					timeSkeleton
+				]),
+				cldr.main([
+					"dates/calendars/gregorian/dateTimeFormats/availableFormats",
+					dateSkeleton
+				])
+			]);
+		}
+		return result;
+	};
```

Comment:
On https://github.com/jquery/globalize/pull/426#discussion_r26713245, I have asked you:

> 2. Given you have created this function, why aren't you using it on case "datetime" in pattern: below? You have duplicate block of code.

You have replied:

> Sir, there is difference in path used in `cldr.main` of blocks of code mentioned. If wanted, we can add a `if` block for `case "datetime" in pattern:` and check for `"datetime" in pattern:` or `"skeleton" in pattern:` to handle both in one function. Should I add?

The only difference is the paths:
- https://github.com/ManrajGrover/globalize/blob/fix-272-handleDateTime/src/date/expand-pattern.js#L40-L41 vs https://github.com/ManrajGrover/globalize/blob/fix-272-handleDateTime/src/date/expand-pattern.js#L95-L96
- https://github.com/ManrajGrover/globalize/blob/fix-272-handleDateTime/src/date/expand-pattern.js#L44-L45 vs https://github.com/ManrajGrover/globalize/blob/fix-272-handleDateTime/src/date/expand-pattern.js#L99-L100

The rest has the same logic. Can you reuse the function somehow (for example, by using additional arguments)?